torch/_torch_docs.py

```diff
@@ -5376,6 +5376,28 @@ def merge_dicts(*dicts):
     tensor([ 1., -1.,  0.,  1.])
 """.format(**common_args))
 
+add_docstr(torch.sgn,
+           r"""
+sgn(input, out=None) -> Tensor
+
+Returns a new complex tensor with the same angle as that of the elements of :attr:`input` and absolute value 1.
+
+.. math::
+    \text{out}_{i} = \frac{{\text{{input}}_i}}{|{\text{{input}}_i}|}
+""" + r"""
+Args:
+    {input}
+    {out}
```

Comment:
Out is an optional kwarg

Reply:
yeah but all the other functions define `out` in a similar way in the doc. also `sgn(input, out=None) -> Tensor` this is indicative of the fact that out is optional kwarg I think